javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -443,111 +441,77 @@ public Object slang_persistent() {
     }
 
     public static class Map extends Base {
-        private final Function<Integer, Integer> MAPPER = i -> i + 1;
+        private static int mapper(int i) { return i + 1; }
```

Comment:
instead of defining a `Functon` and calling `::apply`, where the method expects a different `SAM` (or inlining it, violating `SRP`), a mapper function does both :)